airflow/hooks/oracle_hook.py

```diff
@@ -0,0 +1,79 @@
+import cx_Oracle
+import logging
+
+from airflow.hooks.dbapi_hook import DbApiHook
+
+class OracleHook(DbApiHook):
+    """
+    Interact with Oracle SQL Server.
+    """
+    conn_name_attr = 'oracle_conn_id'
+    default_conn_name = 'oracle_default'
+    supports_autocommit = False
+
+    def get_conn(self):
+        """
+        Returns a oracle connection object
+        """
+        conn = self.get_connection(self.oracle_conn_id)
+        try:
+            conn = cx_Oracle.connect(conn.login, conn.password, conn.host)
+        except:
```

Comment:
Why is this in an except clause? Seems like it should be in an `if`.

You may want to clarify the connection options (especially the extra parameters) in the docstring